INSTALL.md

```diff
@@ -0,0 +1,81 @@
+# Installing the eZ Platform legacy bridge
+
+Unlike eZ Publish 5.x, eZ Platform does not include the Legacy stack by default.i
+Even though it is not officially supported, eZ Publish Legacy can easily be installed
+on top of Platform using Composer.
+
+### Missing legacy extensions
+
+Several ezpublish-legacy extensions are no longer installed by default, such as ezfind.
```

Comment:
Should probably mention `eztags`, too, since it was installed with legacy before.